SwiftificationTests/SequenceTypeTests.swift

```diff
@@ -106,13 +106,16 @@ class SequenceTypeTests: XCTestCase {
             ["man": "Bob"],
             ["woman": "Sally"]
         ]
-        XCTAssertEqual(actual, expected)
+        
+        XCTAssertTrue(actual.elementsEqual(expected, by: { actual, expected in  
```

Comment:
Looks like it can get tighter:

    XCTAssertTrue(actual.elementsEqual(expected) { $0 == $1 })